### DISCUSSION TITLE
Random forest with feature importance

### DISCUSSION THREAD

**Discussion by mathiversen** [smartcorelib/smartcore#270] [Q&A]
Hey!

I'm just playing around with smartcore and ML on my spare time so not entirely sure if this is a relevant question or not. But is it possible to expose the importance of each feature for the random forest output?

So my basic example that im toying with is a price prediction of say a car. How would i go about finding which feature is the most important when deciding on an evaluation of said car?

Thanks in advance!

**Comment by tushushu** [comment]
This issue may be related. https://github.com/smartcorelib/smartcore/issues/124
